src/AbstractServer.php

```diff
@@ -139,15 +127,15 @@ protected function _dispatch(Method\Definition $invokable, array $params)
         $callback = $invokable->getCallback();
         $type     = $callback->getType();
 
-        if ('function' == $type) {
+        if ('function' === $type) {
             $function = $callback->getFunction();
             return call_user_func_array($function, $params);
```

Comment:
This could be refactored to `return $function(...$params);` which is valid in any PHP 7 version.

Reply:
Same here, I would like to provide this change in another PR.